praetorian_cli/sdk/entities/scanners.py

```diff
@@ -0,0 +1,13 @@
+class Scanners:
+
+    def __init__(self, api):
+        self.api = api
+
+    def get(self, key):
+        """ Get scanner details by exact key """
+        return self.api.search.by_exact_key(key)
+
+    def list(self, filter='', offset='', page_size=100):
+        """ List scanners with optional filtering """
+        search_term = filter if filter else '#scanner'
+        return self.api.search.by_term(search_term, None, offset, page_size)
```

Comment:
Does this mean if you run `praetorian chariot list scanners --filter 127.0.0.1` it will not include the key prefix? I assume this is what you meant
```suggestion
        search_term = f"#scanner#{filter}"
        return self.api.search.by_term(search_term, None, offset, page_size)
```